opencensus/pom.xml

```diff
@@ -56,6 +70,10 @@
       <artifactId>opencensus-exporter-stats-stackdriver</artifactId>
       <version>${opencensus.version}</version>
     </dependency>
+    <dependency>
+			<groupId>com.google.cloud</groupId>
```

Comment:
please fix the indentation

Reply:
It looks like it's a formatting issue here in the diff UI. I double-checked in the editor and the formatting is correct. Also, if you look at the "Files changed" tab, it shows up properly. Please LMK if there's something else I can do here.

Reply:
sg, thanks!